charge_lnd/policy.py

```diff
@@ -159,6 +167,78 @@ def match_by_node(self, channel, config):
         if 'node.max_capacity' in config and not config.getint('node.max_capacity') >= node_info.total_capacity:
             return False
 
+        # Consider multiple channels per node policies
+        if set(map(lambda n: "node." + n in config, multiple_chans_props)).issuperset({True}):
```

Comment:
This could be a bit more legible using
`if any(set(map(lambda n: "node." + n in config, multiple_chans_props))):`

Reply:
That is better. Thank you. pylint suggested what I originally had.